importers/nmea_importer.py

```diff
@@ -45,7 +45,9 @@ def can_load_this_header(self, header):
         return "$POSL" in header
 
     def can_load_this_file(self, file_contents):
-        return True
+        # Need to check this isn't WECDIS (Which is derived from this format)
+        contents_string = " ".join(file_contents[0:100])
+        return "VER" not in contents_string and "CHART" not in contents_string
```

Comment:
Minor observation, please @mew-nsc 

There is a chance that `VER` or `CHART` could appear in the first 100 lines, within some other message - particularly as a substring of another piece of text.

But, if tested for `,VER,` and `,CHART,` we could be much more confident that it's not WECDIS data.

Aah, I've just found a `spec` for NMEA.  It demonstrates that `,VER,` and `,CHART,` are not NMEA Messages:
https://gpsd.gitlab.io/gpsd/NMEA.html#_ttm_tracked_target_message

Surprisingly, the spec is maintained by Eric Raymond, the Open Source champion.